### ISSUE TITLE
How to train Faster R-CNN on my own custom dataset and changing the RPN loss functions?

### ISSUE THREAD

**Issue by meshghi** [chenyuntc/Simplified-FasterRCNN#241]
Hello. I want to:

1- train Faster RCNN on my own custom dataset.
2- use the pre-trained Faster RCNN on the VOC2007 as the initial weights to train it then on my own custom dataset.
3- modify the RPN network with different loss functions that I have come up with.

I will greatly appreciate it if you could help me/let me know how to achieve the above goals.

**Comment by meshghi** [comment]
Hello again,

I changed most of the code to work with my custom dataset. But, I'm stuck with handling the "difficult" trining objects. My custom dataset doesn't have any "difficult" object and thus I get errors during training in areas like below:

`difficult.append(int(obj.find('difficult').text))`

I was wondering how I could fix that? Thanks

**Comment by WuXiaoyu55** [comment]
> 

Hello, I encountered the same problem, did you solve it？
difficult.append(int(obj.find('difficult').text))
